src/plugins/data/public/plugin.ts

```diff
@@ -20,16 +20,22 @@
 import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from '../../../core/public';
 import { AutocompleteProviderRegister } from './autocomplete_provider';
 import { DataPublicPluginSetup, DataPublicPluginStart } from './types';
+import { searchService } from './search';
+import { SearchService } from './search/search_service';
```

Comment:
why import both, or why even have searchService function, all it does is `new SearchService(params)` ?

Reply:
Copied the `plugin` way of doing things (this was its own separate plugin so this was the `index.ts` file, at the plugin level, this is  `export plugin...`, here I renamed it `export searchService ...`).  Not necessary except for conformity.  It is pretty silly when you take it outside of the plugin level.  I can remove.